src/NServiceBus.InMemory.PersistenceTests/PersistenceTestsConfiguration.cs

```diff
@@ -4,10 +4,21 @@
     using NServiceBus;
     using NServiceBus.Outbox;
     using NServiceBus.Sagas;
+    using NUnit.Framework;
     using Persistence;
 
     public partial class PersistenceTestsConfiguration
     {
+        //TODO remove, just a demo of potential usage
+        static PersistenceTestsConfiguration()
+        {
+            SagaVariants = new object[]
+            {
+                new TestFixtureData(new TestVariant("just a demo")),
+                new TestFixtureData(new TestVariant("delete me before merging")),
```

Comment:
one downside is that you'd have to manually wrap your variant in a `TestFixtureData` or `object[]` which wasn't necessary with the partial method approach. Maybe this could be tweaked further but not sure it's really worthwhile given the limited expected usage?